cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/SystemArguments.java

```diff
@@ -41,6 +41,21 @@
   private static final String LOG_LEVEL = "system.log.level";
   public static final String TRANSACTION_TIMEOUT = "system.data.tx.timeout";
 
+  public static Map<String, String> getLogLevels(Map<String, String> args) {
+    Map<String, String> logLevels = new HashMap<>();
+    for (Map.Entry<String, String> entry : args.entrySet()) {
+      String key = entry.getKey();
+      if (!key.equals(LOG_LEVEL) && key.startsWith(LOG_LEVEL)) {
```

Comment:
`if (key.length() > LOG_LEVEL.length() && key.startsWith(LOG_LEVEL)) {` to avoid comparing key with LOG_LEVEL twice